backend/src/test/scala/cromwell/backend/standard/callcaching/BlacklistCacheSpec.scala

```diff
@@ -1,22 +1,39 @@
 package cromwell.backend.standard.callcaching
 
 import cromwell.core.CacheConfig
-import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import cromwell.services.CallCaching.CallCachingEntryId
+import org.scalatest.{FlatSpec, Matchers}
 
 import scala.concurrent.duration._
 import scala.language.postfixOps
 
-class BlacklistCacheSpec extends FlatSpec with BeforeAndAfterAll with Matchers {
-  "The blacklist cache" should "default, blacklist and expire" in {
+class BlacklistCacheSpec extends FlatSpec with Matchers {
+  "The blacklist cache" should "default, blacklist, whitelist and expire" in {
+    val hit = CallCachingEntryId(3)
     val bucket = "foo"
-    val cacheConfig = CacheConfig(concurrency = 1, size = Integer.MAX_VALUE, ttl = 1 second)
-    val cache = BlacklistCache(cacheConfig)
-    cache.isBlacklisted(bucket) shouldBe false
+
+    val bucketCacheConfig = CacheConfig(concurrency = 1, size = Integer.MAX_VALUE, ttl = 1 second)
+    val hitCacheConfig = CacheConfig(concurrency = 1, size = Integer.MAX_VALUE, ttl = 1 second)
+    val cache = new RootWorkflowBlacklistCache(bucketCacheConfig = bucketCacheConfig, hitCacheConfig = hitCacheConfig)
+    cache.getBlacklistStatus(bucket) shouldBe Unknown
+    cache.getBlacklistStatus(hit) shouldBe Unknown
     cache.blacklist(bucket)
-    cache.isBlacklisted(bucket) shouldBe true
+    cache.blacklist(hit)
+    cache.getBlacklistStatus(bucket) shouldBe KnownBad
+    cache.getBlacklistStatus(hit) shouldBe KnownBad
 
     // Test ttl
     Thread.sleep(5000L)
```

Comment:
Can we use `eventually` for this, to avoid waiting all the way to the timeout if we can avoid it?